src/hooks/useSearchManager.js

```diff
@@ -72,16 +69,9 @@ export const useSearchManager = () => {
     updateSearchQuery(true)
   }
 
-  // returns true if any filters that are applied, otherwise false
-  const hasAppliedFilters = () => {
-    if (!search) return false
-
-    return (
-      (config.filterOptions || []).filter(
-        (filterOption) => filterOption in search
-      ).length > 0
-    )
-  }
+  const hasNonDownloadableSamples =
+    Number(search[numDownloadableSamples.key]) ===
+    numDownloadableSamples.include
```

Comment:
because this and `hasSelectedFacets` are derived and exported can we move them to the top with the other consts

Reply:
This will be addressed in PR https://github.com/AlexsLemonade/refinebio-web/pull/433.